styled-system/helpers.mjs

```diff
@@ -0,0 +1,280 @@
+// src/assert.ts
```

Comment:
question: I confess I'm surprised the entire `styled-system` is here in the codebase. Is this really intended?